genrl/deep/common/trainer.py

```diff
@@ -368,8 +368,7 @@ def train(self) -> None:
 
             # update params for DQN
             if self.agent.__class__.__name__ == "DQN":
-                if self.agent.replay_buffer.pos > self.agent.batch_size:
-                    self.agent.update_params()
+                self.agent.update_params(timestep)
 
                 if timestep % self.update_interval == 0:
                     self.agent.update_target_model()
```

Comment:
I think this should also be moved inside agent, as its agent-centric. What we could do is define a variable in dqn. like `update_target_interval` and then check inside update_params itself

Reply:
Yea I am doing it